app/views/layouts/_header.html.erb

```diff
@@ -0,0 +1,12 @@
+ <header class="navbar navbar-fixed-top navbar-inverse">
+      <div class="container">
+          <%= link_to "sample app", "#", id: "logo" %>
+          <nav>
+            <ul class="nav navbar-nav navbar-right">
+              <li><%= link_to "Home", "#" %></li>
+              <li><%= link_to "Help", "#" %></li>
+              <li><%= link_to "Log in", "#" %></li>
```

Comment:
Tìm hiểu và sử dụng i18n cho toàn bộ text trên các file html nhé

ngoài ra, file này thiếu end-line. đây là line trống sau cùng ko chứa j, kể cả space
em đọc trên training guideline có code config IDE chỗ này

Reply:
ready